app/com/gu/floodgate/DynamoDBTable.scala

```diff
@@ -71,6 +84,20 @@ trait DynamoDBTable[T] extends StrictLogging {
     getListOfItems(request)
   }
 
+  def getLatestItem(id: String, filterKeyName: String, filterValue: String)(implicit ec: ExecutionContext): Future[Option[T]] = {
+    val request = new QueryRequest().withTableName(tableName)
+      .withScanIndexForward(false)
+      .withKeyConditionExpression(s"$keyName = :$keyName")
```

Comment:
yeah sorry, what I meant is the placeholders are local to the function, but that's a small thing 😅